releasenotes/notes/dbapi2-deprecate-analytics-config-ececae6e9274f852.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    Deprecate `ddtrace.config.dbapi2` as default for `TracedCursor` and
+    `TracedConnection` as well as `DD_DBAPI2_*` environment variables. Use
+    `IntegrationConfig` and `DD_<INTEGRATION>_TRACE_FETCH_METHODS` specific to
```

Comment:
I don't think we should mention IntegrationConfig explicitly since it's not really a public API. I think we mostly refer to it as the "integration config"

Reply:
Might also be worth providing a concrete example with one of the dbapi integrations to demonstrate what the change is here.

Reply:
> I don't think we should mention IntegrationConfig explicitly since it's not really a public API. I think we mostly refer to it as the "integration config"

Ah yes, I forgot to adapt these notes from @majorgreys's original PR! 🙁